.github/workflows/deploy-gcp-tests.yml

```diff
@@ -230,7 +228,10 @@ jobs:
           --ssh-flag="-o ConnectionAttempts=20" \
           --ssh-flag="-o ConnectTimeout=5" \
           --command=' \
-          sudo dmesg \
+          lsblk;
+          sudo lsof /dev/sdb;
```

Comment:
Should this also be `$DISK_NAME`?

Reply:
Yes, @gustavovalverde can we fix the hard-coded sdb in another PR?

Reply:
Fixed at #7709